src/mixins/filing-template-mixin.ts

```diff
@@ -369,7 +367,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // NB: don't parse Name Request object -- NR is fetched from namex/NRO instead
 
     // restore Entity Type
-    this.setEntityType(draftFiling.business.legalType)
+    this.setEntityType(draftFiling.registration.nameRequest.legalType)
```

Comment:
Does this change have to be made for dissolution filings also?

Reply:
No. `business` object is not available for non existing business. In case of  `dissolution`  we have a valid business